### ISSUE TITLE
Create a clear landing page

### ISSUE THREAD

**Issue by Jessseee** [StichtingIAPC/SteamBird#30]
The application needs a clear landing page with information on what the service is used for and an entry point for both BoeCie members or teachers.

**Comment by Jessseee** [comment]
Possible landing page design, similar to IAPC website. Also check out branch [30-landing-page](https://github.com/StichtingIAPC/SteamBird/tree/30-landing-page).
![image](https://user-images.githubusercontent.com/21026046/100667474-d8a99e00-335a-11eb-9c2b-03d11bf967e9.png)

**Comment by rhbvkleef** [comment]
I like how _@luuklsl, Deal with it._ is still in there. A product of ancient times...

I am thinking that a 3-column layout might be more fitting (although more buttons might be added in the future)

**Comment by luuklsl** [comment]
I am wondering what kind of extra buttons you think might be added?

Do like the design, though there is a bit too much of empty space imo.

I am wondering if we want a 'news' feed like box instead of a more static text?
Then again, I do think that easy editing of text is always nice.

**Comment by Jessseee** [comment]
Yeah, an announcement or news section might be nice to add as well. But I wouldn't know what other buttons to add and I think you shouldn't make it too packed with information.

**Comment by rhbvkleef** [comment]
> I am wondering what kind of extra buttons you think might be added?

I was just thinking aloud. If buttons were added, a 3-col layout might become a problem. I have no idea whether we would want to add buttons.

> I am wondering if we want a 'news' feed like box instead of a more static text?
> Then again, I do think that easy editing of text is always nice.

A news feed might be fun, but I think we better postpone that to a later release.
